.github/workflows/build.yml

```diff
@@ -114,6 +114,7 @@ jobs:
         # --no-scan:      For public fork PRs, we won't attempt to publish the scan
         run: |
           ./gradlew --build-cache --info $SCAN_ARG check siteDocTar -x test
+          ./gradlew releaseTarGz
```

Comment:
how about `./gradlew --build-cache --info $SCAN_ARG check siteDocTar releaseTarGz -x test`

Reply:
Nice idea, I will update it.